ec2/sc-ec2-linux-jumpcloud-development.yaml

```diff
@@ -445,7 +446,7 @@ Resources:
                 NAME=$(echo $PRODUCTS | jq '.ProvisionedProducts[0].Name')
                 /usr/bin/aws ec2 create-tags --region $AWS_REGION \
                   --resources $EC2_INSTANCE_ID \
-                  --tags Key=Name,Value=$NAME
+                  --tags Key=Name,Value=$NAME Key=OwnerEmail,Value=$OWNER_EMAIL
```

Comment:
@zaro0508 @ahayden Question: I went ahead and set `OwnerEmail` back on the instance (from the value derived from `aws:servicecatalog:provisioningPrincipalArn`). I was thinking this was a good idea as it's what we've been using. However, I think it's actually preferable to convert to using `aws:servicecatalog:provisioningPrincipalArn` as SC is going to tag that on all the resources it creates, except, evidently, EBS volumes. Thoughts?

Reply:
I think you are correct that `aws:servicecatalog:provisioningPrincipalArn` is the input to use based on how we want SC to work via IAM users *and* assumed role with identity. Is the lack of tagging EBS a bug?